GothicLlamasAPI/src/main/java/com/kainos/ea/objects/JobRoleMapper.java

```diff
@@ -0,0 +1,22 @@
+package com.kainos.ea.objects;
+
+import org.jdbi.v3.core.mapper.RowMapper;
+import org.jdbi.v3.core.statement.StatementContext;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+
+
+public class JobRoleMapper implements RowMapper<CapabilitiesAndRoles> {
+    @Override
+    public CapabilitiesAndRoles  map(ResultSet r, StatementContext ctx) throws SQLException
+    {
+        CapabilitiesAndRoles cap = new CapabilitiesAndRoles();
+        cap.setJobRole(r.getString("jobRole"));
+        return cap;
+    }
+}
+
+
+
```

Comment:
Lots of extra blank lines in this file - most linters will recommend removing them. Also the double space before `map` and the brace on a new line instead of after the method signature.